Simulation.cpp

```diff
@@ -152,15 +152,12 @@ bool Simulation::InitArgs()
     }
 
     if (!_config->GetTrajectoriesFile().empty()) {
-         fs::path p(_config->GetTrajectoriesFile());
-         fs::path curr_abs_path = fs::current_path();
-         fs::path rel_path = _config->GetTrajectoriesFile();
-         fs::path combined = (curr_abs_path /= rel_path);
-         std::string traj = combined.string();
-         _config->SetTrajectoriesFile(traj);
-         if(!fs::exists(traj))
-              fs::create_directories(combined.parent_path());
+         fs::path trajPath(_config->GetTrajectoriesFile());
+         if(!fs::exists(trajPath)) {
```

Comment:
You do that unconditionally, `create_directories` will not fail if the path exists already.